SettingsHelper.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -3,11 +3,11 @@
     "pins": [
       {
         "package": "ButteredToast",
-        "repositoryURL": "git@github.com:Yonodactyl/ButteredToast.git",
+        "repositoryURL": "https://github.com/Yonodactyl/ButteredToast.git",
```

Comment:
It's using https instead of ssh again

Reply:
Oof! Thanks for catching this. SSH fools me again!